build.gradle

```diff
@@ -125,6 +125,8 @@ subprojects {
                 truth                : 'com.google.truth:truth:1.0',
                 slf4jsimple          : 'org.slf4j:slf4j-simple:1.7.25', // Compatibility layer
                 awaitility           : 'org.awaitility:awaitility:3.0.0', // Compatibility layer
+                testcontainers       : 'org.testcontainers:testcontainers:1.13.0',
```

Comment:
Am I missing something. The code is not using `testcontainers`. This could be done without testcontainers. Just generate the collector service and keep the spans in a list. 

This might help you to use testcontainers with Jaeger https://github.com/jaegertracing/jaeger-analytics-java/blob/master/proto/src/test/java/io/jaegertracing/api_v2/GrpcQueryTest.java#L20

Reply:
Hello @pavolloffay, thanks for review. It is using test containers https://github.com/open-telemetry/opentelemetry-java/pull/1038/files#diff-1dea848d3951caf37e6d7ed61b3a7bb5R71
In initial issue https://github.com/open-telemetry/opentelemetry-java/issues/498 @jpkrohling suggested to use testcontainers